CondCore/JetMETPlugins/test/testJetResolutionPayloadInspector.cpp

```diff
@@ -32,39 +32,55 @@ int main(int argc, char** argv) {
   cond::Time_t start = static_cast<unsigned long long>(1);
   cond::Time_t end = static_cast<unsigned long long>(1);
 
-  std::cout << "## Jet Pt Resolution Histograms" << std::endl;
+  edm::LogWarning("JER_PI") << "## Jet Pt Resolution Histograms" << std::endl;
```

Comment:
For test code, not used in productions, there is normally no strict requirement to move out of `std::cout`.
In any case, since you are migrating to `LogWarning`, please add the corresponding header include
```C++
#include "FWCore/MessageLogger/interface/MessageLogger.h"
```
which is now missing